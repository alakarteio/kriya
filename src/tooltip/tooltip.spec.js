/* eslint-disable import/no-extraneous-dependencies, react/jsx-filename-extension */
/* eslint-env jest */
import snap from 'snap'
import Tooltip from './tooltip'

const snapshot = snap(Tooltip)

describe('common/Tooltip', () => {
  describe('graphical (JSX)', () => {
    it('should add custom className', snapshot({ className: 'custom' }))
    it('should add custom style', snapshot({ style: { backgroundColor: 'red' } }))
    it('should add children', snapshot({ children: 'a child' }))
    it('should have a default behaviour', snapshot({}))
    it('should be visible', snapshot({ show: true }))
  })
})

/* eslint-enable import/no-extraneous-dependencies, react/jsx-filename-extension */
