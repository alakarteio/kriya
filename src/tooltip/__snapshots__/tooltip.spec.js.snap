// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/Tooltip graphical (JSX) should add children 1`] = `
<div
  className="tooltip"
  style={Object {}}
>
  a child
</div>
`;

exports[`common/Tooltip graphical (JSX) should add custom className 1`] = `
<div
  className="tooltip custom"
  style={Object {}}
/>
`;

exports[`common/Tooltip graphical (JSX) should add custom style 1`] = `
<div
  className="tooltip"
  style={
    Object {
      "backgroundColor": "red",
    }
  }
/>
`;

exports[`common/Tooltip graphical (JSX) should be visible 1`] = `
<div
  className="tooltip show"
  style={Object {}}
/>
`;

exports[`common/Tooltip graphical (JSX) should have a default behaviour 1`] = `
<div
  className="tooltip"
  style={Object {}}
/>
`;
